src/subdomains/generic/user/models/auth/auth.controller.ts

```diff
@@ -22,7 +22,7 @@ export class AuthController {
   }
 
   @Get('signMessage')
-  getSignMessage(@Query('address') address: string): { message: string; blockchains: Blockchain[] } {
+  getSignMessage(@Query('address') address: string): { message: string; blockchain: Blockchain } {
```

Comment:
I think we still need to return a list of blockchains here, because this address may be used to login for multiple chains.